pkg/pinger/pinger.go

```diff
@@ -46,24 +63,26 @@ var (
 	// Even though we set up the pinger to run continuously, we still have to give it a non-zero timeout else it will
 	// fail so set a really long one.
 	defaultPingTimeout = 87600 * time.Hour
+
+	logger = log.Logger{Logger: logf.Log.WithName("Pinger")}
 )
 
-type PingerInterface interface {
+type Interface interface {
 	Start()
 	Stop()
 	GetLatencyInfo() *LatencyInfo
 	GetIP() string
 }
 
-type PingerConfig struct {
+type Config struct {
 	IP                 string
 	Interval           time.Duration
 	Timeout            time.Duration
 	MaxPacketLossCount uint
 }
 
-type pingerInfo struct {
-	sync.Mutex
+type Info struct {
+	mu                 sync.Mutex
```

Comment:
Maybe we should use 'mutex' instead 'mu' ?

Reply:
it was a suggestion given my linter , may be when the type names where different , now it is happy so leaving it as sync.Mutex

Reply:
It's b/c you had changed the type name to "Info" which exports it.